src/main/java/group1/intern/controller/adminController.java

```diff
@@ -0,0 +1,18 @@
+package group1.intern.controller;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+
+import group1.intern.annotation.PreAuthorizeAdmin;
+
+@Controller
+public class AdminController {

```

Comment:
cái này là model account nhỉ thì nó nên là AccountsController nhé và nằm trong package controller.admin nhé
```suggestion
public class AccountsController {

```